<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Colorizer!</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>400</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
  background-color: white;
border-style: solid;
    border-width: 5px;
    border-color:black;
}</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>400</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="pageMain">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;gowno XD&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pushButton {
 background-color: white;
color: black;
font: 75 18pt &quot;HelveticaNeueLT Std&quot;;
border-style: solid;
    border-width: 5px;
    border-color:black;
 }
 QPushButton#pushButton:pressed {
     background-color: rgb(224, 0, 0);     
 }
 QPushButton#pushButton:hover {
	background-color: rgba(150, 150, 150, 128);
 }</string>
      </property>
      <property name="text">
       <string>encode</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>290</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;gowno XD&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pushButton_2 {
	 background-color: white;
	color: black;
	font: 75 18pt &quot;HelveticaNeueLT Std&quot;;
	border-style: solid;
    border-width: 5px;
    border-color:black;
 }

 QPushButton#pushButton_2:hover {
	background-color: rgba(150, 150, 150, 128);
 }</string>
      </property>
      <property name="text">
       <string>decode</string>
      </property>
     </widget>
     <widget class="QPushButton" name="settingsButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#settingsButton {
 background-color: white;
color: black;
font: 75 24pt &quot;HelveticaNeueLT Std&quot;;
border-style: none;

 }

 QPushButton#settingsButton:hover {
     background-color: rgba(150, 150, 150, 128);
 }</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/logos/question_mark.png</normaloff>
        <normalon>:/logos/question_mark.png</normalon>:/logos/question_mark.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>461</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
    border-width: 0px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/logos/logo_normal.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <zorder>settingsButton</zorder>
     <zorder>pushButton</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>label</zorder>
    </widget>
    <widget class="QWidget" name="pageSettings">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>-10</y>
        <width>191</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(255, 255, 255, 0);
color: black;
font: 75 36pt &quot;HelveticaNeueLT Std&quot;;
border-style: none;</string>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
     <widget class="QPushButton" name="settingsButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#settingsButton_2 {
 background-color: white;
color: black;
font: 75 24pt &quot;HelveticaNeueLT Std&quot;;
border-style: none;

 }

 QPushButton#settingsButton_2:hover {
     background-color: rgba(150, 150, 150,128);
 }</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/logos/back_arrow.png</normaloff>:/logos/back_arrow.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox {
	color: black;
	font: 75 18pt &quot;HelveticaNeueLT Std&quot;;
	border-style: none;
    spacing: 5px;
}

QCheckBox::indicator {
    width: 40px;
    height: 40px;
}

QCheckBox::indicator:unchecked {
    image: url(:/logos/checkb_normal.png);
}

QCheckBox::indicator:unchecked:hover {
    image: url(:/logos/checkb_normal_hover.png);
}

QCheckBox::indicator:unchecked:pressed {
    image: url(:/logos/checkb_normal_pressed.png);
}

QCheckBox::indicator:checked {
    image: url(:/logos/checkb_checked.png);
}

QCheckBox::indicator:checked:hover {
    image: url(:/logos/checkb_checked_hover.png);
}

QCheckBox::indicator:checked:pressed {
    image: url(:/logos/checkb_checked_pressed.png);
}</string>
      </property>
      <property name="text">
       <string>Encode using alpha channel</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox {
	color: black;
	font: 75 18pt &quot;HelveticaNeueLT Std&quot;;
	border-style: none;
    spacing: 5px;
}

QCheckBox::indicator {
    width: 40px;
    height: 40px;
}

QCheckBox::indicator:unchecked {
    image: url(:/logos/checkb_normal.png);
}

QCheckBox::indicator:unchecked:hover {
    image: url(:/logos/checkb_normal_hover.png);
}

QCheckBox::indicator:unchecked:pressed {
    image: url(:/logos/checkb_normal_pressed.png);
}

QCheckBox::indicator:checked {
    image: url(:/logos/checkb_checked.png);
}

QCheckBox::indicator:checked:hover {
    image: url(:/logos/checkb_checked_hover.png);
}

QCheckBox::indicator:checked:pressed {
    image: url(:/logos/checkb_checked_pressed.png);
}</string>
      </property>
      <property name="text">
       <string>Open file after coding</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>340</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(255, 255, 255, 0);
color: black;
font: 75 15pt &quot;HelveticaNeueLT Std&quot;;
border-style: none;</string>
      </property>
      <property name="text">
       <string>Marek Choinski</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
